Algorithms/Introduction/fibonacci_iterative.rb

```diff
@@ -0,0 +1,19 @@
+def fib(n)
+  return nil if n.nil?
+
+  count = 1
+  fib_0 = 0
+  fib_1 = 1
+
+  if n == 0
+    return 0
+  elsif n <= 10
+    while count < (n-1) do
```

Comment:
Another option would be `(0..n-1).each do`. That would avoid the need for `count`.